resource-managers/kubernetes/integration-tests/scripts/setup-integration-test-env.sh

```diff
@@ -58,15 +58,15 @@ while (( "$#" )); do
   shift
 done
 
-if [[ $SPARK_TGZ == "N/A" ]];
+rm -rf $UNPACKED_SPARK_TGZ
+if [[ $SPARK_TGZ == "N/A" && $IMAGE_TAG == "N/A" ]];
```

Comment:
I think this is too strict. You should be able to build images from the build directory too.

Reply:
So @vanzin  is the idea that if neither a SPARK_TGZ or an IMAGE_TAG is specified we'd dynamically build the image on request? Or what do you mean?

Reply:
That would be my expectation.

Having to build a tgz before running the ITs slows down the process. So does manually having to build the images before running the ITs.

If invoking the ITs means "I'll test whatever version of Spark is currently built in your build directory", it speeds up everything.

(Kinda what running the ITs through sbt does.)

Reply:
Sounds reasonable, I'll switch that over.

Reply:
We still want to special case this option I think, but it now runs successfully without an image tag or an release tgz specified. cc @vanzin